src/Response/Directives/Display/Text.php

```diff
@@ -19,4 +19,19 @@ class Text
      * @var string|null
      */
     public $type;
+
+    /**
+     * @param string|null $text
+     * @param string|null $type
+     * @return Text
+     */
+    public static function create($text, $type = self::TYPE_PLAIN_TEXT): Text
+    {
+        $text_ = new self();
```

Comment:
I don't like the underscore naming, perhaps `$text = new self();` here and `public static function create($value = null, $type = self::TYPE_PLAIN_TEXT): Text`